bin/check-node-version.js

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+
+const { execSync } = require( 'child_process' );
+const fs = require( 'fs' );
+const path = require( 'path' );
+
+const currentNodeVersion = process.versions.node;
+const currentMajorNodeVersion = parseInt( currentNodeVersion.split( '.' )[ 0 ] );
```

Comment:
Let's create a utility function to DRY the major version parsing, reducing the line noise a bit to make this easier to scan...

Reply:
Added, please let me know if it looks good now, thank you Tom!